### PR TITLE
Add some quick commands for checking certificates

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:* https://github.com/aws/eks-anywhere-internal/issues/2452

*Description of changes:* We have documented the steps to check certificate expiration from within the nodes, but we had not previously covered how to perform this check from outside the nodes. The newly added steps address this gap, providing a useful method for automated certificate health checks

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/security/manually-renew-certs.md [hunk 1]
```diff
@@ -26,6 +26,16 @@ The following table lists the cluster certificate files:
 |                       | apiserver                |
 |                       | front-proxy-client       |
 
+Commands below can be used for quickly checking your certificates expiring date:
+
+```bash
+# The expiry time of api-server certificate on you cp node
+echo | openssl s_client -connect ${CONTROL_PLANE_IP}:6443 2>/dev/null | openssl x509 -noout -dates
```
Comment (g-gaston) [inline docs/content/en/docs/clustermgmt/security/manually-renew-certs.md:33]: doesn't kubeadm do this already?
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-certs/#cmd-certs-check-expiration

Comment (d8660091) [reply docs/content/en/docs/clustermgmt/security/manually-renew-certs.md:33]: Right, kubeadm can do this and cover more certs for cp nodes (require ssh). I think these quick commands are still worth to note for etcd nodes, double checking (check from outside instead from inside) and adding health-checks.

**Comment by d8660091** [comment]
> Can we update the description details of the PR?

sure

**Pull request merged**